numpy/core/tests/test_function_base.py

```diff
@@ -1,10 +1,10 @@
 from numpy import (
     logspace, linspace, geomspace, dtype, array, sctypes, arange, isnan,
     ndarray, sqrt, nextafter, stack, errstate
-    )
+)
 from numpy.testing import (
     assert_, assert_equal, assert_raises, assert_array_equal, assert_allclose,
-    )
+)
```

Comment:
Style nit - these two changes should be undone - the indentation before the closing parenthesis was correct originally

Reply:
I think this is subjective - I prefer the style in this patch, but the "don't touch code that isn't near your change" argument is strong enough to justify reverting these lines.

Reply:
You're right - my opinion likely stems from `black` rather than pep8

Reply:
Doesn't black advocate the style in this patch?

Reply:
I think you're right - sorry for all the noise!

Reply:
I will revert this part to its original state.